RecoMuon/TrackerSeedGenerator/plugins/TSGForOIDNN.cc

```diff
@@ -391,7 +392,7 @@ void TSGForOIDNN::produce(edm::StreamID sid, edm::Event& iEvent, edm::EventSetup
     hitDoubletSeedsMade = 0;
 
     auto createSeeds = [&](auto const& layers) {
-      for (auto const& layer : layers) {
+      for (auto const& layer : boost::adaptors::reverse(layers)) {
```

Comment:
IIUC, C++20 has 
```suggestion
      for (auto const& layer : layers | std::views::reverse) {
```

@cms-sw/core-l2 do we have C++20 support already in GCC10 and later builds? (now that we stopped building gcc9 in 12_3)

Reply:
@slava77 Trying to compile this results in `'std::views' has not been declared`, so it looks like this is not yet available (or I am missing something).

Reply:
@JanFSchulte 
thank you for checking explicitly

Reply:
> do we have C++20 support already in GCC10 and later builds?

Not yet, but we were thinking to discuss on enabling C++20 early January (and `ranges` is included in libstdc++ of gcc 10)

Reply:
Unfortunately, clang is substantially trailing go in C++20 implementation which will make our migration not so smooth.